portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,58 @@
+#body {
+    background-color: darkturquoise;
+    
+}
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+    width: 90%;
```

Comment:
You should follow a use of whitespace. Consider formatting your CSS to follow [Google's HTML/CSS guide](https://google.github.io/styleguide/htmlcssguide.html). I think you use VSCode? It can probably be configured to automatically format your HTML and CSS to follow the Google style guide.